src/tests/mockdata.js

```diff
@@ -0,0 +1,35 @@
+import dotenv from 'dotenv';
```

Comment:
@NiyoEric I think you can add this in a mock data folder to make our code and folder structure more neater since we might add more mock data files in future.

Reply:
@ivymwende You are right but you have chosen to create it  as a file instead of the folder

Reply:
I have created `mock` directory.